index.html

```diff
@@ -1,429 +1 @@
-
```

Comment:
Doesn't index need the redirect?

Reply:
Why?

Reply:
Or you mean that it should be to xon.sh?

Reply:
Changed: index.html -> xon.sh

Reply:
It's not needed to set the detailed redirects because after time the old pages will go from search index.

Reply:
It would still be good in case people come to this site directly.

Reply:
Don't clearly understand what you want.

Reply:
Just a redirect like the other pages have

Reply:
The `index.html` will be `<html><script>window.location.replace("https://xon.sh/");</script><body>Redirect to https://xon.sh/</body></html>` - like the other pages. What you mean? )

Reply:
Yeah that is what I mean

Reply:
It's already as you want ) Probably Github just hidden the info.

Reply:
oh you are right, thanks